README.md

```diff
@@ -1,24 +1,64 @@
-# README
+# Rails Blog
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> In this project, you will create all controllers and views for your Blog app. As you won't be processing any data yet, you will add simple placeholders for your views with plain HTML.
+## Project requirements
 
```

Comment:
- [Optional] It will be great if you document your project with definite statement of accomplishment and stating what you achieved in you project or what your project is all about for example ....
  "In this project, I create all controllers and views for a blog app...." or "...Describe your project with "...A controller and view app without processing any data, adding placeholder with plain HTML....
  Instead of repeating the project instruction, what do you think?

Reply:
- Also it will be nice when creating PR you do the same next:
 for Example instead of ........."
`Initialize your project with Git.`-----` I initialize this project with Git`
`Important: do NOT use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.`-----`I avoid using rails g scaffold command so as to master controller setup`
`Make sure that your project has Postgres database set up.`-----`I made sure PG db was set up`